index.bs

```diff
@@ -931,19 +931,21 @@ Note: See also the similarly named [=callback function=] definition.
 
 [=Callback interfaces=]
 must not [=interface/inherit=]
-from any non-callback interfaces, and non-callback interfaces must not
+from any interfaces, and non-callback interfaces must not
 inherit from any callback interfaces.
 Callback interfaces must not [=include=] any [=interface mixins=].
 
-[=Static attributes=] and
+[=Regular attributes=],
+[=static attributes=] and
 [=static operations=] must not
 be defined on a [=callback interface=].
 
+[=Callback interfaces=] must define exactly one [=regular operation=].
+
 <div class="advisement">
 
     Specification authors should not define
     [=callback interfaces=]
-    that have only a single [=operation=],
```

Comment:
Are you sure about this deletion?

Reply:
We don't want people to use callback interfaces at all in new APIs, so yes. (And given the other changes, there's no point in having this line, because all callback interfaces only have a single operation.)